packages/p2p/src/components/my-profile/payment-methods/edit-payment-method-form/edit-payment-method-form.tsx

```diff
@@ -54,54 +67,52 @@ const EditPaymentMethodForm = () => {
+                                                value={payment_method_to_edit.display_name}
                                             />
                                         )}
                                     </Field>
-                                    {Object.values(my_profile_store.selected_payment_method_fields).map(
-                                        (payment_method_field, key) => {
-                                            return (
-                                                <Field
-                                                    name={payment_method_field[0]}
-                                                    id={payment_method_field[0]}
-                                                    key={key}
-                                                >
-                                                    {({ field }) => (
-                                                        <Input
-                                                            {...field}
-                                                            data-lpignore='true'
-                                                            error={errors[payment_method_field[0]]}
-                                                            type={
-                                                                payment_method_field[0] === 'instructions'
-                                                                    ? 'textarea'
-                                                                    : payment_method_field[1].type
-                                                            }
-                                                            label={payment_method_field[1].display_name}
-                                                            className={classNames({
-                                                                'add-payment-method-form__payment-method-field':
-                                                                    !errors[payment_method_field[0]]?.length,
-                                                                'add-payment-method-form__payment-method-field--text-area':
-                                                                    payment_method_field[0] === 'instructions',
-                                                            })}
-                                                            onChange={handleChange}
-                                                            name={payment_method_field[0]}
-                                                            required={!!payment_method_field[1].required}
-                                                        />
-                                                    )}
-                                                </Field>
-                                            );
-                                        }
+                                    {selected_payment_method_fields?.map(
```

Comment:
shouldn't this be `Object.values(selected_payment_method_fields)` ?

Reply:
for this no, since selected_payment_method_fields is an array and not an object